terraform/prod.vpc_peering.json

```
@@ -70,5 +70,11 @@
         "account_id": "234617505259",
         "vpc_id": "vpc-08b7d799f1691fd11",
         "subnet_cidr": "172.16.36.0/22"
+    },
+    {
+        "peer_name": "govuk-pay-test",
+        "account_id": "734876687119",
+        "vpc_id": "vpc-031070d650a39111e",
+        "subnet_cidr": "172.17.36.0/22"
```

```suggestion
        "subnet_cidr": "172.16.40.0/22"
```
Can this be continuous within the 172.16 range (ignore govuk-infrastructure-test to see the pattern)

---

Updated - maybe sense check with @rjbaker but I think this is to spec now